streams/shared/src/main/scala/zio/stream/ZTransducer.scala

```diff
@@ -192,6 +192,61 @@ object ZTransducer {
       case None     => ZIO.succeedNow(Chunk.empty)
     }
 
+  /**
+   * Creates a transducer that chunks individual chunks.
+   */
+  def chunkChunks[A]: ZTransducer[Any, Nothing, A, Chunk[A]] =
```

Comment:
```suggestion
  def chunks[A]: ZTransducer[Any, Nothing, A, Chunk[A]] =
```

Reply:
@iravid The name `chunks` could be misunderstood as wrapping individual elements in a chunk.

Reply:
It could, but with proper scaladoc, it won't :-)